scripts/build_twilio_library.py

```diff
@@ -61,6 +62,8 @@ def get_domain_info(oai_spec_location: str, domain: str, is_file: bool = False,
     # added logic to fetch the domain name from servers url in spec file, instead for relying on file name
     if language == 'java' and full_path.endswith('.json'):
         domain_name = parse_domain_name(full_path)
+    if language in {'csharp'}:
+        domain_name = domain_name.capitalize()
```

Comment:
Need to add special rule for `JWT` [domain](https://github.com/twilio/twilio-csharp/tree/main/src/Twilio) name. Since capitalising it would not work.

Reply:
Please ignore, not needed for Domains inside Rest, since there is no special case.